File: HospitalMethod/src/InsuranceCompany.java
import java.util.ArrayList;
import java.util.List;

public class InsuranceCompany {
    public List<Person> clients;
    public DoctorChemicalDrug consultant = new PesimisticDoctorChemicalDrug();

    public InsuranceCompany(List<Person> clients) {
        this.clients = clients;
    }

    public void calculateInsurancePrice()
    {
        for(Person client : clients)
        {
            System.out.println("Client " + client.id + " insurance price is  " + (consultant.calculateMortality(client) * 8));
        }
    }
}

File: HospitalMinix/src/main/scala/InsuranceCompany.scala
class InsuranceCompany(clientc : List[Person]) {
  val clients: List[Person] =  clientc
  val consultant: Consultant = new Consultant();

  def getAllInsurancePrices(): Unit =
  {
    for(client <- clients){
      consultant.calculateInsurancePrice(client)
    }
  }

}
